drivers/hmis/app/models/hmis/form/definition.rb

```diff
@@ -452,4 +452,67 @@ def apply_conditionals(enrollment)
   def project_and_enrollment_match(...)
     instances.map { |i| i.project_and_enrollment_match(...) }.compact.min_by(&:rank)
   end
+
+  # Find and/or initialize CustomDataElements that are collected by this form. Eventually this should be done as part of the
+  # Form Editor admin tool. For now, it is called by a rake task manually.
+  def introspect_custom_data_elements
```

Comment:
We return cdeds, not cdes right? I think it would be better if the method name reflected that
```suggestion
  # Find and/or initialize CustomDataElementDefinitions that are collected by this form. Eventually this should be done as part of the
  # Form Editor admin tool. For now, it is called by a rake task manually.
  def introspect_custom_data_element_definitions
```